src/locale/de/index.js

```diff
@@ -1,24 +1,24 @@
 export default {
-  firstMethod: 'Ich habe ein Array und würde gerne', //'I have an array, I would like to',
-  findMethod: 'Ich möchte folgendes finden', // "I'm trying to find"
+  firstMethod: 'Ich habe einen Array und würde gerne', //'I have an array, I would like to',
+  findMethod: 'Ich möchte', // "I'm trying to find"
   methodOptions: 'Ich muss',
   methodTypes: {
-    add: 'hinzufügen',
-    remove: 'entfernen',
-    find: 'finden',
-    'iterate by': 'durchlaufen',
+    add: '',
+    remove: '',
+    find: '',
+    'iterate by': '',
     'string': 'String zurückgeben'
   },
-  singleItem: 'ein Element', // 'one item',
-  manyItems: 'ein oder mehrere Elemente', // 'one or many items',
+  singleItem: 'ein Element finden', // 'one item',
```

Comment:
this doesn't say find one item in english, it says only 'one item'

Reply:
Yeah, but without the "finden" at this position it doesn't work in German... 
Saying "I would like to find one item" looks in German like this: 
`"Ich möchte ein Element finden"`
In German, infinite verbs (here: "finden") are always at the end of a sentence. Even worse, the English "find" has different translations in German (e. g. "herausfinden", "finden", ...)

Because the `singleItem` and `manyItems` values seem to be only used for the "find", I added it there. I know, it's pretty hacky, but otherwise we'd have to add the option for text after the selector (almost everywhere)...

Reply:
Ohhh I see. My german grammar is terrible, sorry.

Reply:
No problem 😀 localization for German is a difficult task, in my experience, even porting German UIs to English is challenging because of our strange grammar 😀